core/log.cpp

```diff
@@ -81,7 +81,7 @@ void Log::log( const EString &m, Severity s )
         EString n = "Archiveopteryx";
         if ( l )
             n = l->name();
-        fprintf( stderr, "%s: %s\n", n.cstr(), m.simplified().cstr() );
+        fprintf( stderr, "%s: %s\n", n.cstr(), m.cstr() );  // hgu - we want correct output
```

Comment:
Not clear to me. The point of simplified() is generally to avoid multiline loglines; is it certain that m isn't multiline?